### DISCUSSION TITLE
checksum / command not found

### DISCUSSION THREAD

**Discussion by tard44** [toltec-dev/toltec#459] [Support] [answered]
Hello -
I am still having issues installing Toltec, even after the latest update (Stable merge, week 38).
I am running the latest rM2 software  - 2.10.1.332 - and have the latest ddvk patches and the LAMY eraser hack installed. Still, I am receiving this through Terminal -

```terminal
root@reMarkable:~# $ wget http://toltec-dev.org/bootstrap
-sh: $: command not found
root@reMarkable:~# $ echo "2d1233271e0cc8232e86827bcb37ab2a44be2c5675cd15f32952614916ae246a  bootstrap" | sha256sum -c && bash bootstrap
-sh: $: command not found
sha256sum: -: no checksum lines found
```

I am by no means an expert, and so was wondering if you might have some advice?
Thank you very much

**Comment by matteodelabre** [comment, marked as answer]
Hi @tard44, you must omit the leading `$` when copy/pasting commands from the install instructions. This is just a convention used to indicate that what follows is a command to run in a shell, but they are not part of the commands themselves.

**Reply by tard44** [reply]
oh, thank you very much for such a fast response. I've learned something new - told you I was no expert :) Thanks again

**Reply by matteodelabre** [reply]
You’re welcome!
